spec-insert/lib/renderers/templates/example_code.python.mustache

```diff
@@ -0,0 +1,3 @@
+{{! example_code.python.mustache }}
+
+{{{call_code}}}
```

Comment:
The template logic in [example_code_python.rb](https://github.com/opensearch-project/documentation-website/pull/10187/files#diff-c447ebbfe0db5e6184ddbefc2122baaf5c4be96044294581f4e79a6abbe59244) should be put into the mustache template file to help better visualize what the end result would look like. Else we get a 2-line template like this which is not useful.

Not a blocker. You should resolve this in a follow up.